scripts/generated/app.ts

```diff
@@ -1280,6 +1280,7 @@ export const APP_STRING_KEYS = {
 
     // STAKING_REWARDS
 
+    AVAILABLE_IN_DAYS: 'APP.STAKING_REWARDS.AVAILABLE_IN_DAYS',
```

Comment:
hmm it looks like we're missing some keys?`UNSTAKING_FROM` and `STAKING_REWARDS_AVAILABLE`

Reply:
ohh good call, `UNSTAKING_FROM` was duplicated, removing